api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -48,6 +53,9 @@
     @Value("${api.retry-after.delay}")
     private int retryAfterDelay;
 
+    @Value("${api.file-download.path}")
```

Comment:
This is a common property used by both the API and Worker, and it will point at the shared file system mount. So let's name it `efs.mount`